src/main/java/duke/deadline/DeadlineAction.java

```diff
@@ -0,0 +1,32 @@
+package duke.deadline;
+
+import duke.item.Item;
+import duke.exceptions.DukeException;
+import duke.utils.CommandAction;
+import duke.utils.Message;
+import duke.utils.MessageAction;
+
+public class DeadlineAction {
+    /**
+     * Adds a Deadline item into the list.
+     * 
+     * @param parameters Gets the description and deadline of the item.
+     * @throws DukeException when there are missing parameters or areValidParameters check fails
+     */
+    public static Item addDeadline(String parameters, int itemsSize) throws Exception {
```

Comment:
which part of the method adds the item into the list? 
I can see the instantiation but not adding to any list